label_sync/cluster/label_sync_job.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017 The Kubernetes Authors All rights reserved.
```

Comment:
Why do we need Job and CronJob?

Reply:
The job is more useful than a cronjob during development (as essentially a canary)